independent-publisher-connectors/Pexels/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1130 @@
+                          },
+                          "tiny": {
+                            "type": "string",
+                            "description": "The image cropped to W 280px X H 200px."
+                          }
+                        },
+                        "description": "An assortment of different image sizes that can be used to display this Photo."
+                      },
+                      "liked": {
+                        "type": "boolean",
+                        "description": "Tells if the photo is liked by you or not."
+                      },
+                      "alt": {
+                        "type": "string",
+                        "description": "Text description of the photo for use in the alt attribute."
+                      }
+                    }
+                  },
+                  "description": "An array of Photo objects."
+                },
+                "prev_page": {
+                  "type": "string",
+                  "description": "URL for the previous page of results, if applicable."
+                },
+                "next_page": {
+                  "type": "string",
+                  "description": "URL for the next page of results, if applicable."
+                }
+              }
+            }
+          }
+        },
+        "summary": "Search for Photos",
+        "description": "Search Pexels for any photos that you would like. For example your query could be something broad like Nature, Tigers, People. Or it could be something specific like Group of people working.",
+        "operationId": "SearchPhotos",
+        "parameters": [
+          {
+            "name": "query",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The search query.",
+            "x-ms-summary": "Search Query"
+          },
+          {
+            "name": "orientation",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Desired photo orientation.",
+            "x-ms-summary": "Orientation",
+            "enum": [
+              "landscape",
+              "portrait",
+              "square"
+            ]
+          },
+          {
+            "name": "size",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Minimum photo size.",
+            "x-ms-summary": "Size",
+            "enum": [
+              "large",
+              "medium",
+              "small"
+            ]
+          },
+          {
+            "name": "color",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Desired photo color. Supported colors: red, orange, yellow, green, turquoise, blue, violet, pink, brown, black, gray, white or any hexidecimal color code (eg. #ffffff).",
```

Comment:
All descriptions must be free from grammar and syntax errors. There is a typo in this description ("hexidecimal").

Reply:
Corrected.